pkgs/development/tools/fnlfmt/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib, stdenv, fetchFromSourcehut, fennel, lua }:
+
+stdenv.mkDerivation rec {
+  pname = "fnlfmt";
+  version = "0.2.1";
+
+  src = fetchFromSourcehut {
+    owner = "~technomancy";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-JIqeQhI3fFGrej2wbj6/367IZqWAFegySc2R8IDmvGE=";
+  };
+
+  nativeBuildInputs = [ fennel ];
+
+  buildInputs = [ lua ];
+
+  buildPhase = ''
+    runHook preBuild
+
+    echo "#!${lua}/bin/lua" > fnlfmt
+    ${fennel}/bin/fennel --require-as-include --compile cli.fnl >> fnlfmt
+    chmod +x fnlfmt
+
+    runHook postBuild
+  '';
+
+  installPhase = ''
+    runHook preInstall
+
+    patchShebangs ./fnlfmt
```

Comment:
Why do we need to patch the shebang?

Reply:
Ah you're right, this is no longer necessary since we're not using the Makefile.